<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello Qt World!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Quit Application</string>
    </property>
    <property name="text">
     <string>Quit!</string>
    </property>
   </widget>
   <widget class="QLabel" name="helloLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="counterLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="counterPlusBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Increment Counter</string>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="counterMinusBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Decrement Counter</string>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Show Calculator</string>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
